doc/ref_model/chapters/chapter03.rst

```diff
@@ -787,139 +776,217 @@ Storage Scenarios and Architecture Fit
--  "N" - No, not available
--  "NA" - Not Applicable for this Use Case / Stereotype
-
-============================ =================================== ================== ==== === =================== ==================== =========== ===== === ==== ======
-\                                                                                            Tenant / User
-============================ =================================== ================== ==== === =================== ==================== =========== ===== === ==== ======
-\                                                                Infra / Ctrl / Mgt          Platform Native                          Shared File                Object
-Use Case                     Stereotype                          Boot               Ctrl Mgt Hypervisor Attached Container Persistent Within      Cross Ext vNAS Object
-Data-centre Storage          Dedicated Network Storage Appliance Y                  Y    Y   Y                   Y                    O           O     O   O    O
-\                            Dedicated Software Defined Storage  O                  O    O   Y                   Y                    O           O     O   O    O
-\                            Traditional SAN                     Y                  Y    Y   N                   N                    N           N     N   N    N
-Satelite data-centre Storage Small Software Defined Storage      O                  O    O   Y                   Y                    O           O     O   O    O
-Small data-centre Storage    Converged Software Defined Storage  O                  O    O   Y                   Y                    O           O     O   O    O
-Edge Cloud                   Edge Cloud for VNF/CNF Storage      NA                 O    NA  Y                   Y                    O           O     O   O    O
-\                            Edge Cloud for Apps Storage         NA                 O    NA  Y                   Y                    O           O     O   O    Y
-\                            Edge Cloud for Content Mgt Storage  NA                 O    NA  Y                   Y                    O           O     O   O    Y
-============================ =================================== ================== ==== === =================== ==================== =========== ===== === ==== ======
+- "Y" - Yes and almost always provided
+- "O" - Optional and readily accommodated
+- "N" - No, not available
+- "NA" - Not Applicable for this Use Case / Stereotype
++------------------------------+-------------------------------------+------+------+------+------------+------------+--------+-------+-----+------+--------+
+|                                                                                         | Tenant / User                                                  |
++==============================+=====================================+======+======+======+============+============+========+=======+=====+======+========+
```

Comment:
Add + only to where the line intesects with an other one.

Reply:
@CsatariGergely tables render correctly and having the markers (+) help with ensuring alignment across merged cells.